server/controllers/stock/lots.js

```diff
@@ -92,6 +94,76 @@ async function update(req, res, next) {
   }
 }
 
+/**
+ * GET /lot_dupes/:label?/:inventory_uuid?/:initial_quantity?/:entry_date?/:expiration_date?
+ *
+ * @description
+ * Returns all lots with the given label or matching field(s)
+ * inventory_uuid, initial_quantity, entry_date, expiration_date
+ *
+ * TODO: After getting this working purge unneeded params
+ *
+ */
+function getDupes(req, res, next) {
+  const options = db.convert(req.query, ['inventory_uuid']);
+  const filters = new FilterParser(options, { tableAlias : 'l' });
+  filters.fullText('label');
+  filters.equals('inventory_uuid');
+  filters.equals('initial_quantity');
```

Comment:
Looks like you have put `initial_quantity` in twice.  Go ahead and remove it.

Reply:
Ooops. Fixed.